### DISCUSSION TITLE
Does `pattern` work for anyone?

### DISCUSSION THREAD

**Discussion by huyz** [vsls-contrib/codetour#289] [Q&A]
Not only does setting `recordMode` to `pattern` instead of `lineNumber` not seem to do anything but lines, but it interferes with adding a new step (from within an editor) in Edit mode.

**Comment by skaparelos** [comment]
Not sure if this repo is  maintained anymore. I found a fix. See the beginning of the [README file in this repo](https://github.com/skaparelos/codetour/tree/main?tab=readme-ov-file#fix-pattern-mode-on-mac-perhaps-works-in-other-oss-as-well)
